### ISSUE TITLE
BUG[]It seems  wrong

### ISSUE THREAD

**Issue by qidouhai** [marticliment/UniGetUI#2480]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions.
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose)).

### UniGetUI Version

3.0.2

### Windows version, edition, and architecture

win 10  盗版的

### Describe your issue

如下，这反馈还需要填写这么多东西啊。好繁琐，钩子随便√了

### Steps to reproduce the issue

1

### UniGetUI Log

```text
1Starting package install operation for package id=9NBLGGH419H3with Manager name=Winget
Given installation options are <InstallationOptions: SkipHashCheck=False;InteractiveInstallation=False;RunAsAdministrator=False;Version=;Architecture=;InstallationScope=;InstallationScope=;CustomParameters=;RemoveDataOnUninstall=False>
Process Executable     : F:\Program Files\WingetUI\PackageEngine\Managers\winget-cli_x64\winget.exe
Process Call Arguments :  install --id 9NBLGGH419H3 --exact --accept-source-agreements --silent --disable-interactivity --accept-package-agreements
Working Directory      : C:\Users\admin
    |    | 
    | Failed when searching source: winget
    | An unexpected error occurred while executing the command: 
    | 0x80070005 : �ܾ����ʡ�
    | The following packages were found among the working sources.
    | Please specify one of them using the --source option to proceed.
    | Name        Id           Source
    | --------------------------------
    | 360安全卫士 9NBLGGH419H3 msstore
Process Exit Code      : -2147024891
Process End Time       : 2024/7/12 15:57:34
```

### Package Managers Logs

```text
都在上面呢1
```

### Relevant information

1

### Screenshots and videos

1

**Comment by marticliment** [comment]
Please install UniuGetUI 3.1.1 beta 1 (https://github.com/marticliment/UniGetUI/releases/download/3.1.1-beta1/UniGetUI.Installer.exe)

**Comment by marticliment** [comment]
Fixed by 3.1.1-beta1
